datasets/ua/war_sanctions/crawler.py

```diff
@@ -0,0 +1,362 @@
+from zavod import Context, helpers as h
+
+
+LINKS = [
+    # {  # child kidnappers
+    #     "url": "https://war-sanctions.gur.gov.ua/en/kidnappers/persons?page=1&per-page=12",
+    #     "type": "person",
+    # },
+    # {  # child kidnappers
+    #     "url": "https://war-sanctions.gur.gov.ua/en/kidnappers/companies?page=1&per-page=12",
+    #     "type": "company",
+    # },
+    # {  # russian athletes
+    #     "url": "https://war-sanctions.gur.gov.ua/en/sport/persons?page=1&per-page=12",
+    #     "type": "person",
+    # },
+    {  # ships
+        "url": "https://war-sanctions.gur.gov.ua/en/transport/ships?page=1&per-page=12",
+        "type": "vessel",
+    },
+]
+
+# TODO: c/o and uknowns, Невідомо, 'Unknown (22.03.2024), Ship Safety Management Manager (IMO / Country / Date)': 'Unknown (08.02.2023)
+
+
+def lookup_override(context, key):
+    override_res = context.lookup("overrides", key)
+    if not override_res:
+        context.log.warning(f"No override found for {key}")
+        return {}
+
+    extracted = {}
+    for override in override_res.items:
+        if override["prop"] == "name":
+            extracted["name"] = override["value"]
+        elif override["prop"] == "registrationCode":
+            extracted["registrationCode"] = override["value"]
+
+    if not extracted:
+        context.log.warning(f"No matching properties found in overrides for {key}")
+
+    return extracted
+
+
+def extract_label_value_pair(label_elem, value_elem, data):
+    label = label_elem.text_content().strip().replace("\n", " ")
+    value = value_elem.text_content().strip().replace("\n", " ")
+    value = " ".join(value.split())
+    # we use it for splitting some strings at a later stage
+    value = " | ".join(
+        [text.strip() for text in value_elem.itertext() if text.strip()]
```

Comment:
instead of _this join to later split_ thing, it'd be more standard to make the type if `value` be `str | List[str]` if simply `List[str]` isn't sufficient. `Entity.add` does the right thing with `List[str]` - no looping required so no logic needed for the different types